test/e2e/cpu_assignment_test.go

```diff
@@ -254,7 +310,7 @@ func makeTesterPodWithExclusiveCPUClaim(ns, image string, cpuClaim *resourcev1.R
 
 	return &v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: "tester-pod-excl-cpu-claim-",
+			GenerateName: "pk-tester-pod-excl-cpu-claim-",
```

Comment:
guess this is a debug value?

Reply:
Fixed in https://github.com/kubernetes-sigs/dra-driver-cpu/pull/16/commits/979a29cc0a6e12f95c89b8f17a2fa3288c4399cd